src/posix/main.c

```diff
@@ -194,6 +205,18 @@ static void ParseArg(int aArgCount, char *aArgVector[], PosixConfig *aConfig)
         case OT_POSIX_OPT_RADIO_VERSION:
             aConfig->mPrintRadioVersion = true;
             break;
+#if __linux__
+        case OT_POSIX_OPT_REAL_TIME_SIGNAL:
+            if (optarg[0] == '+')
+            {
+                aConfig->mPlatformConfig.mRealTimeSignal = SIGRTMIN + atoi(&optarg[1]);
+            }
+            else
+            {
+                aConfig->mPlatformConfig.mRealTimeSignal = atoi(optarg);
```

Comment:
I checked the previous code, the micro timer signal is always set to `OPENTHREAD_CONFIG_MICRO_TIMER_SIGNAL` and is by default set to `SUGRTMIN`.  And it is determined at compile time. Now it is determined at run-time. What is the consideration of this? I think the signal won't change after the program starts to run.
And shall we ensure that the assigned signal number doesn't collide with other signals? Like the interrupt signal. Will it cause some problems?

Reply:
If we want to distribute OpenThread POSIX as a library, we should allow users to pass in this value, which can be compile time defined or run-time given in the application code.